scrapers/nus-v2/package.json

```
@@ -53,6 +54,7 @@
     "date-fns": "2.16.1",
     "deep-diff": "1.0.2",
     "fs-extra": "9.0.1",
+    "he": "^1.2.0",
```

Ditto above. Also run `yarn` to fix the lock file after these changes 

```suggestion
    "he": "1.2.0",
```

---

fixed in 0bee168